spec/board_spec.rb

```diff
@@ -42,4 +42,11 @@
     end
   end
 
+  context 'when a single token occupies all given positions on the board' do
+    it 'returns true' do
+      board = Board.new(layout: %w[X X X 4 5 6 7 8])
+      result = board.found_winner_at(winning_combination: [0, 1, 2], token: 'X')
+      expect(result).to be_truthy
```

Comment:
Here we're expecting any value that is truthy which doesn't quite match what the descrption of this test, perhaps we can change it to:
```ruby
expect(result).to eq true
```